x86_symbolic/src/Data/Macaw/X86/Crucible.hs

```diff
@@ -755,7 +756,7 @@ uDivRem sym repsz macawNum1 macawNum2 macawDenom =
     -- Get numerator
     numExt <- getNumerator dw sym macawNum1 macawNum2
     -- Get denominator
-    den <- getDenominator dw sym macawDenom
+    den <- getDenominator dw sym "div" macawDenom
```

Comment:
There's something a bit funny going on here. The error message specifically mentions the `div` instruction, but the name of this function (`uDivRem`) describes itself as performing a general unsigned division operation. Given that we are now encoding the name of the instruction itself in the error message, I wonder if we should just update the Haddocks to say that this is meant to power the `div` instruction in particular.

(Relatedly, the Haddocks claim that it does not handle remainders, but this appears to be out of date with the implementation.)